src/models/Settings.js

```diff
@@ -19,22 +19,17 @@ define([
   'util/BrowserFeatures',
   'util/Util',
   'util/Logger',
+  'util/IDP',
   'config/config.json'
-],
-function (Okta, Q, Errors, BrowserFeatures, Util, Logger, config) {
+], function (Okta, Q, Errors, BrowserFeatures, Util, Logger, IDP, config) {
 
   var SharedUtil = Okta.internal.util.Util;
-
+  var _ = Okta._;
```

Comment:
Hi, just curious as to why we are not using ES6 (let/const) through out the file. Even if this file is not transpiled looks like IE11 supports let / const (as long as we don't use a for loop in which case the behavior could be different when not transpiled).

Reply:
this part of old code.

Reply:
there is no reason to not use `let/const` but trying to be consistent in the old code.